### ISSUE TITLE
`lang="en"` can get missing

### ISSUE THREAD

**Issue by yezhengli-Mr9** [fhightower/html-to-json#21]
For following, even `lang="en"` can get missing
```html
<html lang="en"><head>
        <meta charset="utf-8">
        <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate">
        <meta http-equiv="Pragma" content="no-cache">
        <meta http-equiv="Expires" content="0">
        ...
        </head>
        <body></body>
 </html>
```

**Comment by fhightower** [comment]
Good catch - thanks for reporting. Unfortunately, I don't have time to work on this at the moment without [sponsorship](https://github.com/sponsors/fhightower). If you're using this python package for profit, please consider sponsoring me. Thanks!

Regardless, I'll try to swing back to this project sometime before the end of the year.

**Comment by fhightower** [comment]
@yezhengli-Mr9 : I see that you closed this issue. Is this still a bug?

**Comment by fhightower** [comment]
Going to reopen this unless I hear that this is not a bug.

**Comment by fhightower** [comment]
@yezhengli-Mr9 : I'm not able to reproduce, so I'm going to close this issue. But feel free to reopen if it comes up again.
